features/snippet.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Snippet life cycle
+
+    Scenario: Creating a snippet
+        Given I am authenticated
+        When I fill in the snippet creation form
```

Comment:
Nice, maybe you could add one more step before this one where you'll go to the new snippet page (even if you have that in this step, just to be clear from where you're creating snippet).
P.S. generally we're using two spaces for indentation in all files